.teamcity/src/main/kotlin/common/Os.kt

```diff
@@ -55,14 +55,15 @@ enum class Os(
 
     fun asName() = name.lowercase().toCapitalized()
 
-    fun javaInstallationLocations(): String {
+    fun javaInstallationLocations(arch: Arch = Arch.AMD64): String {
```

Comment:
🤔 It looks like we should have an `Agent` abstraction that brings together `Os` and `Arch`. This would then be a good place to have functions like `javaInstallationLocations`.